### ISSUE TITLE
About the accuracy computation(关于计算准确率的问题)

### ISSUE THREAD

**Issue by shiyanpei** [auroua/InsightFace_TF#100]
Hi, I found the implementation of accuracy just take the logits and then pass it into a softmax.

```
pred = tf.nn.softmax(logit)
acc = tf.reduce_mean(tf.cast(tf.equal(tf.argmax(pred, axis=1), labels), dtype=tf.float32))
```
However, the computation of the logits make use the ground truth labels. So how can this be true? 

Thank you.

你好，我发现计算准确率的实现是用arcface 算法生成的logits 直接进行softmax函数计算的， 但是再arcface 算法中，logits 的生成需要用到数据的真实label。所以测试集的准确率是不是不可以这样计算？
谢谢

**Comment by 18335100284** [comment]
+1
